room.py

```diff
@@ -106,19 +154,59 @@ def allowgames(bot, cmd, room, params, user):
     return reply.response('{param} is not a supported parameter'.format(param = params))
 
 def tour(bot, cmd, room, params, user):
+    """ Independent command for initiating tours in this room.
+   
+    This is only possible for rooms where this bot has at least '@' rank. Intended
```

Comment:
This depends on the room to be honest, since different rooms can set this to a different setting. Historically it was # or @, but some months ago % was made possible to start tours too. So this is only sorta-accurate, but I can't even check the rank required on load, so I don't know what to do regarding it.